server/handlers/componetgenratorhelper.go

```diff
@@ -425,17 +415,21 @@ func ErrMsgContruct(response *models.RegistryAPIResponse) string {
 	msg += ")"
 	return msg
 }
+
 func determinePluralWord(count int, word string) string {
```

Comment:
Such logic should reside in `meshkit` alongside other helper functions like formatToCamelCase....